kyo-scheduler-zio/jvm/src/main/scala/kyo/KyoSchedulerRuntime.scala

```diff
@@ -0,0 +1,31 @@
+package kyo
```

Comment:
perhaps `kyo.zio` namespace?

Reply:
I think a `zio` package in Kyo's codebase could get confusing since the packages can clash if in the same scope.

Reply:
Good point

Reply:
I've renamed the object to from `KyoSchedulerRuntime` to `KyoSchedulerZioRuntime`. It should help avoid confusion since the previous name could indicate it's a runtime for Kyo itself.